cloud/azure/deploy/down.go

```diff
@@ -25,25 +25,6 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-type DownStreamMessageWriter struct {
```

Comment:
Is this intentional?

Reply:
these should be using the common message writer now